docs/docker-machine-drivers.md

```diff
@@ -1,18 +1,23 @@
-# Driver plugin installation
+# Docker Machine driver installation
```

Comment:
I was thinking about this as well and yes the underlying library is libmachine and we don't use 'docker-machine' as binary/dependency, but the drivers are still known as Docker Machine Drivers - https://docs.docker.com/machine/drivers/. Searching for "libmachine driver" does not give the results you want, but "Docker Machine driver" does.